bots/pr/src/main/java/org/openjdk/skara/bots/pr/CommandHandler.java

```diff
@@ -30,6 +30,11 @@
 import java.util.List;
 
 interface CommandHandler {
-    void handle(PullRequestBot bot, PullRequest pr, CensusInstance censusInstance, Path scratchPath, String args, Comment comment, List<Comment> allComments, PrintWriter reply);
+    void handle(PullRequestBot bot, PullRequest pr, CensusInstance censusInstance, Path scratchPath, CommandInvocation command, List<Comment> allComments, PrintWriter reply);
     String description();
+
+    default boolean multiLine() {
+        return false;
+    }
+    default boolean allowedInBody() { return false; }
```

Comment:
```suggestion
    default boolean allowedInBody() {
        return false;
    }
```

Suggestion to be consistent with style :wink: